specification/draft/index.html

```diff
@@ -905,89 +905,95 @@ <h3>Verifiable Presentation of DSCSA Stakeholder Credentials</h3>
       <p>
         Verifiable Presentations may be used to combine and present credentials. They can be packaged in such a way that
```

Comment:
> Verifiable Presentations may be used to combine and present credentials. They can be packaged...
I don't quite get the purpose of the intro. The "may" & "can" are non-normative. So, OCI doesn't seem to state that VP are to be used (is there an alternative?) but then goes on to define VP structure and errors. Or have I missed a clear statement of the requirement elsewhere?

Reply:
I have not looked at the other parts of this spec in detail but I agree with you.

Reply:
I just exchanged the may with are